websites/S/Streamlabs/presence.ts

```diff
@@ -80,33 +101,84 @@ presence.on("UpdateData", async () => {
 				case false:
 					presenceData.smallImageKey = "pause";
 					presenceData.smallImageText = strings.pause;
-					presenceData.endTimestamp = null;
+					presenceData.endTimestamp = new Date(
+						Date.now() + (video.duration - video.currentTime) * 1000
+					).getTime();
+					break;
+				default:
+					presenceData.smallImageKey = null;
+					presenceData.smallImageText = null;
 					break;
 			}
 			presenceData.details = `Watching ${clipTitle[0]} to`;
 			presenceData.state = clipTitle[2];
 		} else if (document.location.pathname.includes("/dashboard")) {
 			switch (document.location.hash) {
+				case "#/":
+					titleDashboard = `${userType[1]}Dashboard`;
+					break;
+				case "#/stats":
+					titleDashboard = `${userType[1]}Stats`;
+					break;
+				case "#/advancedstats":
+					titleDashboard = `${userType[1]}Stats`;
+					break;
+				case "#/recentevents":
+					titleDashboard = `${userType[1]}Recent Events`;
+					break;
+				case "#/alertbox":
+					titleDashboard = `${userType[1]}Alert Box`;
+					break;
 				case "#/widgets":
 					titleDashboard = `${userType[1]}Widgets`;
 					break;
-				case "#/cloudbot/mod-tools":
+				case getLocationHash("/cloudbot"):
 					titleDashboard = `${userType[1]}Cloudbot`;
 					break;
-				case "#/stats":
-					titleDashboard = `${userType[1]}Dashboard`;
+				case getLocationHash("/multistream"):
+					titleDashboard = `${userType[1]}MultiStream`;
 					break;
-				case "#/alertbox":
-					titleDashboard = `${userType[1]}Alert Boxes`;
+				case getLocationHash("/prime"):
```

Comment:
again, if it doesn't contain "/prime" this will create a case for "", thus making all of these cases for "" which is redundant, you should get rid of this function IMO